### PR TITLE
feat: added support for go-stgin

### PR DESCRIPTION
**Pull request by AminMal**

### CODE REVIEW

File: go/stgin/config.yaml [hunk 1]
```diff
@@ -0,0 +1,3 @@
+framework:
+  website: github.com/AminMal/stgin
+  version: 1.1.0
```
Comment (waghanza) [inline go/stgin/config.yaml:3]:
```suggestion
  version: 1.1
```

File: go/stgin/config.yaml [hunk 2]
```diff
@@ -0,0 +1,3 @@
+framework:
+  website: github.com/AminMal/stgin
```
Comment (waghanza) [inline go/stgin/config.yaml:2]:
```suggestion
  github: AminMal/stgin
```

File: go/stgin/go.mod [hunk 3]
```diff
@@ -0,0 +1,7 @@
+module main
+
+go 1.18
```
Comment (waghanza) [inline go/stgin/go.mod:3]:
```suggestion
```

File: go/stgin/go.mod [hunk 4]
```diff
@@ -0,0 +1,7 @@
...
+
+require github.com/AminMal/stgin v1.1.0
+
+require github.com/AminMal/slogger v1.0.0 // indirect
```
Comment (waghanza) [inline go/stgin/go.mod:7]: why both ? is there no a require only module that require both ?

**Review by waghanza — CHANGES_REQUESTED** [review]

File: go/stgin/go.mod (hunk 4 above)
Comment (AminMal) [reply go/stgin/go.mod:7]: Unfortunately not for now. That's the logging lib. But I'll make sure to take a look on how to do it in future stgin releases.

**Review by waghanza — APPROVED** [review]

**Pull request merged**